### DISCUSSION TITLE
A smaller, fully-assembled version with USB-C

### DISCUSSION THREAD

**Discussion by ak66666** [hrvach/deskhop#128] [Show and tell]
I used the v1.0 schematics to create a more compact version without RPi Pico but with RP2040 instead, and USB-C instead of micro-USB.
I called it Deskhop Jr.

Instead of the fully-3D printed case I used an empty PCB connected with the standoffs. Later I added a wrap-around wall to have somewhat better component protection. The exposed PCB side got very few powered lines, and I covered them with the silkscreen, so they should be protected well enough.
LEDs signal the active side - green for the right and red for the left. They are bright enough even at the modest current to shine through the translucent PLA wall.

For the next revision I made minor layout changes, primarily to move the components away from the board edges. Shorter stand-offs will be used, so it will be a tad slimmer. Holes above the reset buttons are added. Less RAM to be used (16MB is an overkill, but that's what I used from the QMK schematics, so maybe 2 or 4, or 8, whichever is available and cheaper to manufacture.)
Also I plan to make them in black, as I realize that red is not everyone's favorite color. It may make the holes in PCBs less conspicuous.

![IMG_20240724_170451028](https://github.com/user-attachments/assets/58d217dd-d461-4dff-8828-9324dde340d8)
![IMG_20240724_170502788](https://github.com/user-attachments/assets/c6ce3357-57f3-437c-8d60-e2434afd3fdd)
![IMG_20240724_170522690](https://github.com/user-attachments/assets/1a1878ed-8c7f-4e49-a9b2-c75d73f20b6e)
![IMG_20240724_170402191_HDR](https://github.com/user-attachments/assets/92879d2a-1616-4dc6-8144-0e1410924f7b)

**Comment by hrvach** [comment]
This looks **super cool**, I love it! Are the case decorations filament swap and a layer on top?

**Reply by ak66666** [reply]
Thank you.
Case is decorated with silkscreen print.  (edit: the top cover part just an empty PCB with the same print as the bottom of the "active" PCB.)
The only 3D printed part is the wall between the boards.

**Comment by walklinewang** [comment]
@ak66666 兄弟，这个东西的链接可以发给我吗？我试了按图搜索没找到M3的，按关键字搜索又找不到一模一样的

![image](https://github.com/user-attachments/assets/6f00dd78-9b44-4cbd-a921-8eb24ab96dfc)

**Reply by ak66666** [reply]
https://www.digikey.ca/en/products/detail/keystone-electronics/9021/2746026

**Reply by walklinewang** [reply]
Thanks, I found it~

**Comment by Scalpos** [comment]
Wow @ak66666 it's super cute and compact 👍 
Are the board and case schematics available on GitHub ?

**Reply by ak66666** [reply]
Thank you.
No, I haven't published it on GitHub yet, selling them on Etsy.

**Reply by mujkalaszlo** [reply]
are you selling the whole assembly on etsy ?

**Reply by ak66666** [reply]
Yes, completely assembled: https://www.etsy.com/ca/listing/1763041320/deskhop-jr-a-keyboard-and-mouse-switch

**Reply by mujkalaszlo** [reply]
Can I buy it directly from you or should I go through etsy ?

**Reply by ak66666** [reply]
Go through Etsy please.

**Reply by mujkalaszlo** [reply]
cool, thanks :)

**Comment by omelettedufromagee** [comment]
Can the Jr. version be updated with the same firmware as the "standard version"?

**Reply by ak66666** [reply]
Yes, it uses the very same firmware as the original Deskhop model.
it is effectively the same set of electronic components put on a single PCB.
